interface/usergroup/usergroup_admin_add.php

```diff
@@ -35,621 +35,62 @@
-    <span><?php echo xlt('Save'); ?></span></a>
-<a class="btn btn-link btn-cancel" id='cancel' href='#'>
-    <span><?php echo xlt('Cancel');?></span>
-</a>
-</td></tr></table>
-<br /><br />
-
-<table border='0'>
-
-<tr>
-<td valign='top'>
-<form name='new_user' id="new_user" method='post' action="usergroup_admin.php">
-<input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
-
-<input type='hidden' name='mode' value='new_user'>
-<input type='hidden' name='secure_pwd' value="<?php echo attr($GLOBALS['secure_password']); ?>">
-
-<span class="font-weight-bold">&nbsp;</span>
-<table class="border-0" cellpadding='0' cellspacing='0' style="width:600px;">
-<tr>
-    <td colspan="4">
-        <?php
-        // TODO: we eventually want to move to a responsive layout and not use tables here.  So we are going to give
-        // module writers the ability to inject divs, tables, or whatever inside the cell instead of having them
-        // generate additional rows / table columns which locks us into that format.
-        $preRenderEvent = new UserEditRenderEvent('usergroup_admin_add');
-        $GLOBALS['kernel']->getEventDispatcher()->dispatch($preRenderEvent, UserEditRenderEvent::EVENT_USER_EDIT_RENDER_BEFORE);
-        ?>
-    </td>
-</tr>
-<tr>
-<td style="width:150px;"><span class="text"><?php echo xlt('Username'); ?>: </span></td><td style="width:220px;"><input type="text" name="rumple" style="width:120px;" class="form-control"><span class="mandatory"></span></td>
-<?php if (empty($GLOBALS['gbl_ldap_enabled']) || empty($GLOBALS['gbl_ldap_exclusions'])) { ?>
-<td style="width:150px;"><span class="text"><?php echo xlt('Password'); ?>: </span></td><td style="width:250px;"><input type="password" style="width:120px;" name="stiltskin" class="form-control"><span class="mandatory"></span></td>
-<?php } else { ?>
-        <td><input type="hidden" value="124" name="stiltskin" /></td>
-<?php } ?>
-</tr>
-<tr>
-    <td style="width:150px;"></td><td style="width:220px;"></td>
-    <td style="width:200px;"><span class='text'><?php echo xlt('Your Password'); ?>: </span></td>
-    <td class='text' style="width:280px;"><input type='password' name=adminPass style="width:120px;"  value="" autocomplete='off' class="form-control"><font class="mandatory"></font></td>
-
-</tr>
-<tr>
-<td><span class="text"<?php echo ($GLOBALS['disable_non_default_groups']) ? " style='display: none'" : ""; ?>><?php echo xlt('Groupname'); ?>: </span></td>
-<td>
-<select name="groupname" class="form-control"<?php echo ($GLOBALS['disable_non_default_groups']) ? " style='display:none'" : ""; ?>>
-<?php
-$res = sqlStatement("select distinct name from `groups`");
-$result2 = array();
-for ($iter = 0; $row = sqlFetchArray($res); $iter++) {
-    $result2[$iter] = $row;
+function isValidEmail(email) {
+    return /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email);
 }
 
-foreach ($result2 as $iter) {
```

Comment:
Why did you remove all this code? It doesn't look like you moved it anywhere... you just removed it.  That makes no sense to me.  What are you doing here?